README.md

```diff
@@ -142,15 +144,16 @@ In order to update all the vendored libraries in the `/vendor` directory.
 
 Once you've made the update to the codebase that you'd like to submit, you may [create a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) to the opentelemetry-php project.
 
-After you open the pull request, the CI/CD pipeline will run all of the associated [github actions](https://github.com/open-telemetry/opentelemetry-php/actions/workflows/php.yml).
+After you open the pull request, the CI/CD pipeline will run all the associated [github actions](https://github.com/open-telemetry/opentelemetry-php/actions/workflows/php.yml).
 
-You can simulate the important github actions locally before you submit your PR by running the following command:
+You can simulate the important GitHub actions locally before you submit your PR by running the following command:
 
 ```bash
 make all
 ```
+Note: Copying `.env.dist` to `.env` fixes the warning ```The "PHP_USER" variable is not set. Defaulting to a blank string.```
```

Comment:
I think this is really a necessary setup step - can we move it somewhere prominent? I think the very first step after git clone?

Reply:
yes, @brettmc, I agree. I wanted to do that but was slightly hesitant. I think it'll be placed better after the `git clone`